libopenfpga/libfpgaaesencryption/src/AESfunctions.h

```diff
@@ -0,0 +1,17 @@
+#include <iostream>
+using namespace std;
+
+// These are AES constants for AES 128, 192, 256
+const unsigned short Nb = 4; // columns (could be changed to a larger number in the future)

```

Comment:
For constants, we typically use all capitalize names, such as ``NB``.
The wave naming such as ``CipherKey`` is for data structures.

Also the constants limits the flexibility of using the encryption algorithm. We do not want to change the constant each time we compile the c++ files. I would recommend to rework the functions so that they can accept the constants as input arguments.
For example,
```
void AES_Encrypt(unsigned int* plaintext,
                             unsigned char* expandedKey,
                             unsigned short Nr,
                             unsigned char* ciphertext,
                             short Nb);

```